src/routes/posts.js

```diff
@@ -38,7 +38,7 @@ const fetchPosts = catchAsync( async(req, res) => {
 			})})
 		),
 		metadata: {
-			totalCount: feed.length
+			totalCount: Number(count.count)
```

Comment:
kindly handle this in the service before returning it, that's where the main logic is and if we want count to be a number, we should define it there. 

Else wherever we call the same function we would have to do that

Reply:
yes please